### PR TITLE
[plugin.audio.sverigesradio] 3.0.0

### PR DESCRIPTION
**Pull request by dodoadoodoo**
### Description
Update to support matrix/python 3
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.audio.sverigesradio/README [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (sarbes) [inline plugin.audio.sverigesradio/README:1]: An empty file?!

File: plugin.audio.sverigesradio/addon.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        <import addon="xbmc.python" version="3.0.0" />
+        <import addon="script.module.xbmcswift2" version="19.0.5"/>
+        <import addon="script.module.requests" version="2.25.1"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <summary lang="en">Swedish National Radio (Sveriges Radio)</summary>
```
Comment (sarbes) [inline plugin.audio.sverigesradio/addon.xml]: Wrong language identifiers.

File: plugin.audio.sverigesradio/resources/settings.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```
Comment (sarbes) [inline plugin.audio.sverigesradio/resources/settings.xml:1]: Just an FYI, this style might me outdated in the future: https://kodi.wiki/view/Add-on_settings_conversion

**Review by sarbes — CHANGES_REQUESTED** [review]

File: plugin.audio.sverigesradio/README (hunk 1 above)
Comment (dodoadoodoo) [reply plugin.audio.sverigesradio/README:1]: Yeah, I got nothing to say :-) Do you recommend that I remove or should I add some boilerplate?

File: plugin.audio.sverigesradio/addon.xml (hunk 2 above)
Comment (dodoadoodoo) [reply plugin.audio.sverigesradio/addon.xml]: Fixed

File: plugin.audio.sverigesradio/resources/settings.xml (hunk 3 above)
Comment (dodoadoodoo) [reply plugin.audio.sverigesradio/resources/settings.xml:1]: Thanks for the heads-up, will redo at some point in time

**Comment by ronie** [comment]
looks good to me, thx!

**Pull request merged**